app/views/clients/status.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="client-status">  
   <h1>
-    Application Status for: <strong class="text-primary"><%= @client.full_name %></strong><br />
+    <%= I18n.t "client_status.application_status_for" %><strong class="text-primary"><%= @client.full_name %></strong><br />
     <small>
-      Thank you for your interest in LUCHA's services. See the steps below to complete your application.
+      <%= I18n.t "client_status.thank_you" %>
     </small>
```

Comment:
I think we just need to add translations for the employee section that you added.